packages/components/modules/activity-log/LogGroups/types.ts

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react'
+
+import { LoadMoreFn } from 'react-relay'
+import { VirtuosoProps } from 'react-virtuoso'
+import { OperationType } from 'relay-runtime'
+
+export interface LogGroupsProps {
+  logGroups: LogGroup[]
+  LoadingState?: FC
+  LoadingStateProps?: any
+  VirtuosoProps?: Partial<VirtuosoProps<any, any>>
+  loadNext: LoadMoreFn<OperationType>
+  hasNext: boolean
+  isLoadingNext: boolean
+}
+export interface LogGroup {
+  lastActivityTimestamp: string
+  logs: Log[]
+}
+
+export interface Log {
```

Comment:
lets avoid casting our own type based on a query, we can grab it from the query interfaces, like:

```
// under `packages/components/modules/activity-log/types.ts`
export type ActivityLogs = NonNullable<ActivityLogsFragment$data['activityLogs']>
export type ActivityLogEdges = ActivityLogs['edges']
export type ActivityLogNode = NonNullable<ActivityLogEdges[number]>['node'] // --> this would be your Log
```